### DISCUSSION TITLE
I can haz text-to-speach driver for F5-TTS

### DISCUSSION THREAD

**Discussion by spiraloid** [griptape-ai/griptape#1526] [Ideas]
- [x] I have read and agree to the [contributing guidelines](https://github.com/griptape-ai/griptape#contributing).

it would be switfty nifty to have this open source TTS as a text-to-speech driver.

https://swivid.github.io/F5-TTS/

even cooler if I could just record the 15 second voice style prompt directly in comfy.

**Comment by collindutter** [comment]
Hey @spiraloid this we can try supporting this model via a `HuggingFacePipelineTextToSpeechDriver` assuming it is supported by [transformers TTS](https://huggingface.co/docs/transformers/en/tasks/text-to-speech#text-to-speech).

I'll see if this is easy to wire up and get back to you.
